src/psyclone/tests/domain/common/algorithm/alg_psyir_test.py

```diff
@@ -296,12 +301,7 @@ def test_aic_createpsylayersymbolrootnames():
     assert invoke._psylayer_routine_root_name is None
     assert invoke._psylayer_container_root_name is None
 
-    routine_name = "invoke_0_kern"
-    routine_symbol = invoke.psylayer_routine_symbol
-    assert isinstance(routine_symbol, RoutineSymbol)
-    assert routine_symbol.name == routine_name
-    container_symbol = routine_symbol.interface.container_symbol
-    assert container_symbol.name == f"{routine_name}_mod"
+    invoke.create_psylayer_symbol_root_names()
```

Comment:
What's the reason for this change?

Reply:
According to `git blame` that was Rupert doing this, not me. I'd guess a side effect of merging in the latest master??

Reply:
OK.